src/styles/main.css

```diff
@@ -86,26 +91,30 @@
         }
     }
 
+     /* Home background image */
+    .bg-illustration {
+        background-repeat: no-repeat;
+        background-position: bottom left;
+        background-size: contain;
+    }
+
     @media (max-width: 768px) {
         .container.bg-illustration {
-            padding-bottom: 15.63rem !important; /* Equivalent to 250px */
+            padding-bottom: 15.63rem !important;
```

Comment:
Ah No, ill go ahead and remove that.

Reply:
Generally you'll want to avoid the `!important` when possible.  The main reason is it makes it difficult to override later.  When you use `!important` it even takes precedence over inline `style` attributes.